qiskit/providers/aer/noise/errors/readout_error.py

```diff
@@ -243,7 +244,7 @@ def _check_probabilities(probabilities, threshold):
             if abs(sum(arr) - 1) > threshold:
                 raise NoiseError("Invalid probabilities: sum({})= {} "
                                  "is not 1.".format(vec, sum(arr)))
-            if len(arr[arr < 0]) > 0:
+            if arr[arr < 0]:
```

Comment:
Numpy arrays can't be checked for non-empty like lists, this will give a ValueError if `arr[arr > 0]` is non-empty.

It should be `if arr[arr < 0].size:` or  `if arr[arr < 0].size > 0:`

A couple of lines above could also be changed to `len(arr)` -> `arr.size` if you don't want to use `len`